src/ontology/mondo-edit.obo

```diff
@@ -271426,6 +271422,7 @@ is_a: MONDO:1040055 {source="https://clinicalgenome.org/working-groups/clinical-
 intersection_of: MONDO:0019200 ! retinitis pigmentosa
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/10254 ! ROM1
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/9942 ! PRPH2
+relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/mondo/qc-multiple-gene-associations.sparql
```

Comment:
I don't think this is correct. 
According to [OMIM](https://www.omim.org/entry/608133?search=608133&highlight=608133), this record refers to the disease for which the RDS gene (PRPH2) is mutated, however, "A digenic form of retinitis pigmentosa resulting from a mutation in the RDS gene ([179605.0004](https://www.omim.org/entry/179605#0004)) and a null mutation of the ROM1 gene (see [180721.0001](https://www.omim.org/entry/180721#0001)) has been reported."

Therefore, for this record, we cannot say that in 100% of the case, the disease is caused by a mutation in both genes (or in a mutation in only one gene). 

The gene annotations should be removed for this disease. 

Note: the definition for this disease should also be changed to make this disease concept not specific to the mutation in the PRPH2 gene.

Reply:
Thanks Sabrina. I agree and will make the changes.

Reply:
Updated